sample-project/package.json

```diff
@@ -29,7 +29,7 @@
     "automergeType": "branch-push"
   },
   "devDependencies": {
-    "babel-eslint": "8.0.2",
+    "babel-eslint": "8.1.2",
```

Comment:
seems like this incremented the minor, not the patch, @rarkins

Reply:
Wow, now THAT's a bug. Will look at it now

Reply:
OK, not as bad as I'd thought. It's happening because GitHub's annoying bug where apps cannot automerge when certain branch protections are set. But as a result the app is skipping a bit of code where it should be updating. I'll create an issue in the Renovate repo for this to fix.

Reply:
Fix on the way

Reply:
Thanks!

Reply:
All done. Thanks for the heads up!

Reply:
done with pleasure 😄